src/components/sequencing/actions/ActionRunCard.svelte

```diff
@@ -79,4 +105,15 @@
     display: flex;
     gap: 8px;
   }
+
+  .action-run-wrapper {
+    position: relative;
+  }
+
+  .cancel-button {
+    position: absolute;
```

Comment:
I would avoid absolutely positioning the button, because it takes it out of the flow of the rest of row and it'll make vertically centering inflexible. What you can do is make the containing element a `display: flex` and then adding `justify-content: space-between`.